lib/pure/os.nim

```diff
@@ -11,10 +11,10 @@
 ## retrieving environment variables, reading command line arguments,
 ## working with directories, running shell commands, etc.
 
-runnableExamples("-r:off"):
+runnableExamples:
   let myFile = "/path/to/my/file.nim"
   assert splitPath(myFile) == (head: "/path/to/my", tail: "file.nim")
-  assert parentDir(myFile) == "/path/to/my"
+  assert parentDir(myFile) == "/path/to/my".unixToNativePath
   assert splitFile(myFile) == (dir: "/path/to/my", name: "file", ext: ".nim")
   assert myFile.changeFileExt("c") == "/path/to/my/file.c"
```

Comment:
But now it's confusing that the other examples do not use `.unixToNativePath`. And examples must be simple. These are not unit tests here, they are examples that we ensure keep working.

Reply:
> These are not unit tests here, they are examples that we ensure keep working.

The example is broken on windows.

When users on windows copy the example, running it, finding it not work, they will be more surprised. 
How about removing it or using
```nim
  when defined(posix):
    assert parentDir(myFile) == "/path/to/my"
  else:
    assert parentDir(myFile) == "\\path\\to\\my"
```

Reply:
`when defined(posix):` is ok, but still a bit weird for example code...